pkgs/applications/networking/testssl/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, fetchFromGitHub, pkgs, lib }:
```

Comment:
Please do not pass all of `pkgs`, instead declare each parameter you need directly. Also note that `lib` is typically accessed via `stdenv.lib`.